pkg/gui/gui_config.go

```diff
@@ -0,0 +1,15 @@
+package gui
+
+import gui_server "github.com/Kong/kuma/pkg/config/gui-server"
+
+type GuiConfig struct {
+	ApiUrl      string `json:"apiUrl"`
+	Environment string `json:"environment"`
+}
+
+func FromServerConfig(config gui_server.GuiConfig) GuiConfig {
```

Comment:
Let's keep `FromServerConfig()` func in a package other than `types/` (e.g., in `server/` rather than `server/types/`)

Motivation: to make it clear that REST API doesn't depend on config types